main.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/extservice-operator/flag"
+	"github.com/giantswarm/extservice-operator/server"
+	"github.com/giantswarm/extservice-operator/service"
+)
+
+const (
+	notAvailable = "n/a"
+)
+
+var (
+	description = "The extservice-operator manages connection from the guest cluster pods to the guset cluster services."
```

Comment:
guest

Reply:
I also think it is `connections`, plural.